fairer-iOS/fairer-iOS/Screens/Group/GroupMain/GroupMainViewController.swift

```diff
@@ -11,19 +11,21 @@ import SnapKit
 
 final class GroupMainViewController: BaseViewController {
     
-    let userName: String = "고가혜"
-    
+    private var userName:String = String() {
```

Comment:
p4;
```suggestion
    private var userName: String = String() {
```
띄어쓰기 부탁드립니다,,, ㅎㅎ

Reply:
[cc9e6af](https://github.com/fairer-iOS/fairer-iOS/pull/136/commits/cc9e6afeef76294d44564276b41b5b42b4c31c48)